### ISSUE TITLE
[clsim] Bug in applyDetectorSim.py (Trac #1021)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1015]
I 'm currently working on the standard candle simulation. My task is to integrate the newer noise generator Vuvuzela into the program in place of the older generator caled I3NoiseGeneratorModule. The scripts I'm working with include the following:

generateTestFlashes.py
applyCLSim.py
applyDetectorSim.py
generateTestFlashes.py

The bug lies in applyDetectorSim.py. This script should run correctly when we integrated Vuvuzela in it. However, after discussing it with others on the icecube software slack chat, we discovered that the script doesn't work when we integrated Vuvuzela because it wasn't updated in the past. It uses an out-of-date data class I3MCHIT, rather than the now-used I3MCPE.

Migrated from https://code.icecube.wisc.edu/ticket/1021
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:15:08", 
    "description": "I 'm currently working on the standard candle simulation. My task is to integrate the newer noise generator Vuvuzela into the program in place of the older generator caled I3NoiseGeneratorModule. The scripts I'm working with include the following:\n\ngenerateTestFlashes.py\napplyCLSim.py\napplyDetectorSim.py\ngenerateTestFlashes.py\n\nThe bug lies in applyDetectorSim.py. This script should run correctly when we integrated Vuvuzela in it. However, after discussing it with others on the icecube software slack chat, we discovered that the script doesn't work when we integrated Vuvuzela because it wasn't updated in the past. It uses an out-of-date data class I3MCHIT, rather than the now-used I3MCPE.", 
    "reporter": "vesaw", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067308113782", 
    "component": "combo simulation", 
    "summary": "[clsim] Bug in applyDetectorSim.py", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-06-17T16:16:00", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150810T22:13:21: 
* olivas changed priority from "normal" to "blocker"
* olivas changed milestone from "" to "IceSim 4.2"
* olivas changed status from "new" to "accepted"
* olivas changed owner from "" to "olivas"

**Comment by icecube-trac** [comment]
Trac update at 20150915T05:50:11: 
* olivas changed status from "accepted" to "assigned"
* olivas changed owner from "olivas" to "claudio.kopper"
* olivas changed summary from "Bug in applyDetectorSim.py" to "[clsim] Bug in applyDetectorSim.py"

**Comment by icecube-trac** [comment]
Trac update at 20150915T06:06:59: olivas commented:

> Needs to switch to the python version of I3TimeShifter since the C++ version was removed.

**Comment by icecube-trac** [comment]
Trac update at 20150922T01:40:00: olivas changed milestone from "IceSim 4.2" to "IceSim 5.0"

**Comment by icecube-trac** [comment]
Trac update at 20151104T22:09:49: 
* claudio.kopper changed priority from "blocker" to "normal"
* claudio.kopper commented:

> this is definitely not a blocker. It needs an expert on current detector simulation to check this. Since this is neither a standard script not a test, I will downgrade the priority of this ticket.

**Comment by icecube-trac** [comment]
Trac update at 20151104T22:14:57: claudio.kopper changed milestone from "IceSim 5.0" to "IceSim 5.1"

**Comment by icecube-trac** [comment]
Trac update at 20170327T19:36:24: 
* claudio.kopper commented:

> Clearly not going to happen for this release cycle, postponing to IceSim 6.


* claudio.kopper changed milestone from "IceSim 5.1" to "IceSim 6"

**Comment by icecube-trac** [comment]
Trac update at 20180613T13:02:58: olivas changed owner from "claudio.kopper" to "olivas"

**Comment by icecube-trac** [comment]
Trac update at 20180625T18:42:49: 
* olivas changed resolution from "" to "fixed"
* olivas changed status from "assigned" to "closed"
* olivas commented:

> In changeset 163656:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="163656"
> updated script to use vuvuzela and ony DOMLauncher.  closes #1015.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:15:08: 
* olivas commented:

> Milestone IceSim 6 deleted


* olivas changed milestone from "IceSim 6" to ""
